File: stresser.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class PingLoop {
    public static void main(String[] args) {
        try {
            BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
            System.out.print("Enter IP Address: ");
            String ipAddress = reader.readLine();

            while (true) {
                Process process = Runtime.getRuntime().exec("ping -c 1 " + ipAddress);
                int exitCode = process.waitFor();
                if (exitCode != 0) {
                    System.out.println("Ping failed for " + ipAddress);
                } else {
                    System.out.println("Ping successful for " + ipAddress);
                }
            }
        } catch (IOException | InterruptedException e) {
            e.printStackTrace();
        }
    }
}

File: stresser.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val reader = BufferedReader(InputStreamReader(System.`in`))
    print("Enter IP Address: ")
    val ipAddress = reader.readLine().trim()

    while (true) {
        val process = ProcessBuilder("ping", "-c", "1", ipAddress).start()
        val exitCode = process.waitFor()
        if (exitCode == 0) {
            println("Ping successful for $ipAddress")
        } else {
            println("Ping failed for $ipAddress")
        }
    }
}
